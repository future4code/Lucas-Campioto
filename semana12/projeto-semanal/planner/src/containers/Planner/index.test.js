import React from 'react'
import renderer from "react-test-renderer";
import {Planner} from './index';

describe ('teste componente task', () => {
    it('renders correctly', () => {
        const tree = renderer
            .create(<Planner></Planner>)
            .toJSON();
        expect(tree).toMatchSnapshot(); 
    })
})