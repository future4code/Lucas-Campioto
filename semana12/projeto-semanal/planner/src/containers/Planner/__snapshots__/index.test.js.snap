// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`teste componente task renders correctly 1`] = `
<div>
  <div
    className="sc-bwzfXH dSEICL"
  >
    <h3>
      Tarefas
    </h3>
    <form>
      <div>
        <input
          label="Tarefa"
          name="text"
          onChange={[Function]}
          required={true}
          type="text"
          value=""
        />
      </div>
      <div
        className="sc-bxivhb fFUfNw"
      >
        <select
          name="day"
          onChange={[Function]}
          required={true}
        >
          <option
            disabled={true}
            selected={true}
            value=""
          >
            Dia da Semana
          </option>
          <option
            value="segunda"
          >
            Segunda-feira
          </option>
          <option
            value="terca"
          >
            Terça-feira
          </option>
          <option
            value="quarta"
          >
            Quarta-feira
          </option>
          <option
            value="quinta"
          >
            Quinta-feira
          </option>
          <option
            value="sexta"
          >
            Sexta-feira
          </option>
          <option
            value="sabado"
          >
            Sábado
          </option>
          <option
            value="domingo"
          >
            Domingo
          </option>
        </select>
      </div>
      <button
        onClick={[Function]}
        type="submit"
      >
        Criar Tarefa
      </button>
    </form>
    <div
      className="sc-bdVaJa eIffcY"
    >
      <div>
        <div>
          <h3>
            Segunda
          </h3>
        </div>
      </div>
      <div>
        <div>
          <h3>
            Terça
          </h3>
        </div>
      </div>
      <div>
        <div>
          <h3>
            Quarta
          </h3>
        </div>
      </div>
      <div>
        <div>
          <h3>
            Quinta
          </h3>
        </div>
      </div>
      <div>
        <div>
          <h3>
            Sexta
          </h3>
        </div>
      </div>
      <div>
        <div>
          <h3>
            Sabádo
          </h3>
        </div>
      </div>
      <div>
        <div>
          <h3>
            Domingo
          </h3>
        </div>
      </div>
    </div>
  </div>
</div>
`;
